message_definitions/v1.0/messages.xml

```diff
@@ -2647,6 +2647,12 @@
     <field name="data" type="uint8[]"/>
   </message>
 
+ <message name="EXTERNAL_MAGNETO_RAW" id="154" link="forwarded">
```

Comment:
Maybe name it `EXTERNAL_MAG_RAW` analogous to `IMU_MAG_RAW`?
And fix indentation...